docs/countme.md

```diff
@@ -25,17 +25,25 @@ standalone way.
 
 ## Disabling DNF Count Me on a system
 
-To disable this feature, you need to update the Count Me option in all RPM
-repository configuration files:
+To disable this feature, you need to stop and disable the
+`rpm-ostree-countme.timer` and mask the corresponding units as a precaution:
 
 ```
-sed -i 's/countme=1/countme=0/g' /etc/yum.repos.d/*.repo
+$ systemctl disable --now rpm-ostree-countme.timer
+$ systemctl mask rpm-ostree-countme.service rpm-ostree-countme.timer
 ```
 
-You should also stop and disable the `rpm-ostree-countme.timer`:
+If you have packages layered on top of the base image from an RPM repository,
+then you will have to make sure that the `countme` option is disabled there
+until we fix [this issue in libdnf](TODO):
 
 ```
-$ systemctl disable --now rpm-ostree-countme.timer
+sed -i 's/countme=1/countme=0/g' /etc/yum.repos.d/*.repo
 ```
 
+Note that once you do that, those repository configuraion files won't be
+tracked by ostree/rpm-ostree anymore thus you will have to update them manually
+when major Fedora version rebase happen such as moving from Fedora 33 to Fedora
+34 for example.
```

Comment:
```suggestion
Note that once you do that, those repository configuration files will be locally
modified, and ostree will hence ignore any other changes to the defaults.
```

Reply:
In particular, this bit 
>  you will have to update them manually when major Fedora version rebase happen such as moving from Fedora 33 to Fedora 34 for example.

I think isn't accurate because the repo files use `$releasever` - and rpm-ostree will actually pick up that value from the new tree when rebasing (and when upgrading).  In fact correctly handling `$releasever` in this way is a lot of the reason we take the latency hit of "Checking out base tree" on operations.

Reply:
Pushed a slightly modified version to mention where the default config is stored.